### ISSUE TITLE
BioC2021 check: github action workflow(s)

### ISSUE THREAD

**Issue by kevinrue** [cmirzayi/EpiForBioWorkshop2021#1]
Hi Chloe,

Just doing the rounds of the workshops, listed on https://bioc2021.bioconductor.org/workshops-status/ for the BioC2021 committee.

I notice that you have disabled the default GitHub Action workflow and set up a separate one.

The workflow that works (`pkgdown`) seems to only deploy the pkgdown website.
The template workflow that you have disabled was designed to also build and push a Docker image to DockerHub, which now doesn't happen.

Did you set up an automated build on Docker Hub separately? https://hub.docker.com/r/cmirzayi/epiforbioworkshop2020 seems to have images.

Bottom line is that the badge tracking your workshop on the BioC2021 website (link above) can be changed to point to your `pkgdown` workflow, with the caveat that it will not be representative of the status of your Docker image build.

Let me know what you think!
Kevin

**Comment by cmirzayi** [comment]
Hi Kevin,

Yes I was giving the workshop to another group and kept getting errors from the action so I switched over to the pkgdown workflow for that talk. My intent is to get it working soon with the default Github Action workflow. I will work on it this week.

Thanks,
Chloe

**Comment by kevinrue** [comment]
OK, thanks!

When you're ready to try, I would recommend deleting all your current workflows and simply taking the current version of Sean's template here:
https://github.com/seandavi/BuildABiocWorkshop/blob/master/.github/workflows/basic_checks.yaml

When I tested it myself, I only needed to change line 5, which should be the name of 
your DockerHub repository (in your case, this seems to be `cmirzayi/epiforbioworkshop2020` though you might want to make another one for 2021:
https://github.com/seandavi/BuildABiocWorkshop/blob/master/.github/workflows/basic_checks.yaml#L5

**Comment by cmirzayi** [comment]
The action just rant successfully so I think we're good to go. Thanks for your help.
